bccsp/sw/ecdsa_test.go

```diff
@@ -176,6 +177,13 @@ func TestEcdsaPublicKey(t *testing.T) {
 	invalidCurve := &elliptic.CurveParams{Name: "P-Invalid"}
 	invalidCurve.BitSize = 1024
 	k.pubKey = &ecdsa.PublicKey{Curve: invalidCurve, X: big.NewInt(1), Y: big.NewInt(1)}
+	defer func() {
+		if p := recover(); p != nil {
```

Comment:
there is a function [require.Panics](https://pkg.go.dev/github.com/stretchr/testify/require#Panics), let's use it instead.

Reply:
you means like this ?
```go
require.Panics(t, func() {
    _, err = k.Bytes()
})
```

but it would be failed when it is not version 1.19. If now we need support both 1.17 and 1.19 ?

Reply:
Then let's use `runtime.Version`